protos/class.proto

```diff
@@ -0,0 +1,66 @@
+syntax = "proto3";
+
+import "protos/status.proto";
+import "protos/quiz.proto";
+import "protos/chat.proto";
+import "protos/user.proto";
+
+service PresenterClassService {
+    rpc CreateClass (ClassCreationRequest) returns (ClassCreationCallback);
+    rpc UpdateClass (ClassUpdateRequest) returns (Status);
+    rpc DeleteClass (ClassDeleteRequest) returns (Status);
+
+    rpc StartClass (ClassStartRequest) returns (stream ChatMessage);
```

Comment:
nie rozumiem, dlaczego StartClass zwraca ChatMessage?

Reply:
ponieważ przez jakiś kanał muszą do niego docierać pytania od uczniów

Reply:
poza tym mogą być to inne stuct'y, więc można to zrobić podobnie jak po stronie ucznia - czyli jakiś odpowiednik ClassEvent